.github/workflows/e2e_cloud.yml

```diff
@@ -0,0 +1,92 @@
+name: E2E Cloud
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - staging
+      - trying
+  pull_request:
+    branches: [main]
+    paths:
+      - "**"
+      - "!/*.md"
+      - "!/**.md"
+
+jobs:
+  cloud_e2e_tests:
+    name: Cloud E2E (${{ matrix.os }}) ${{ matrix.environment }}.
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        environment: [Production, Development]
+
+    env:
+      RUST_BACKTRACE: full
+
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v3
+
+      - name: Create Fluvio k3d Cluster
+        uses: AbsaOSS/k3d-action@v2.3.0
+        with:
+          cluster-name: "fluvio"
+          args: --timeout 5m
+
+      - name: Ensure k3d cluster is ready
+        run: |
+          sleep 5
+          ./actions/k3d-check.sh
+
+      - name: Install Fluvio Local Cluster
+        timeout-minutes: 10
+        uses: infinyon/fluvio@master
+        with:
+          cluster-type: k3d
+
```

Comment:
maybe we should be explicit about `stable` version

Reply:
Got it! Pushing!